API.md

```diff
@@ -345,3 +347,31 @@ Queues thumbnail generation tasks for all non-root folders created between `begi
   bannerImage: string (URL)
 }
 ```
+
+## Billing
+
+### POST /gift
```

Comment:
It looks like this is `billing/gift` -- even though this is in the "Billing" section I think above we put the whole path for things like `archive/featured`. Can you use `billing/gift` here to be consistent?